### PR TITLE
Scripting of Aluve App

### PR DESCRIPTION
**Pull request by MlulekiN**

### CODE REVIEW

File: src/test/java/stepDefinition/BookNowTopMenuSteps.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package stepDefinition;
+
+import cucumber.TestContext;
+import io.cucumber.java.en.Then;
+import org.junit.Assert;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import pages.BookNowPage;
+
+public class BookNowTopMenuSteps {
```
Comment (benedictnkosi) [inline src/test/java/stepDefinition/BookNowTopMenuSteps.java]: have one file for menu steps

File: src/test/java/stepDefinition/BookNowTopMenuSteps.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    TestContext testContext;
+    WebDriver driver;
+
+    public BookNowTopMenuSteps(TestContext context) {
+        testContext = context;
+        driver = testContext.getWebDriverManager().getDriver();
+    }
+
+    @Then("User validates content form is displayed")
```
Comment (benedictnkosi) [inline src/test/java/stepDefinition/BookNowTopMenuSteps.java]: what is content form?

Comment (benedictnkosi) [reply src/test/java/stepDefinition/BookNowTopMenuSteps.java]: Do you meaning the booking form?

File: src/test/java/stepDefinition/BookNowTopMenuSteps.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    public void user_validates_content_form_is_displayed() {
+        WebElement bookNowContentPage = driver.findElement(BookNowPage.contentForm);
```
Comment (benedictnkosi) [inline src/test/java/stepDefinition/BookNowTopMenuSteps.java]: what is bookNowContentPage

**Review by benedictnkosi — CHANGES_REQUESTED** [review]

File: src/test/java/stepDefinition/BookNowTopMenuSteps.java (hunk 2 above)
Comment (MlulekiN) [reply src/test/java/stepDefinition/BookNowTopMenuSteps.java]: Done

File: src/test/java/stepDefinition/BookNowTopMenuSteps.java (hunk 1 above)
Comment (MlulekiN) [reply src/test/java/stepDefinition/BookNowTopMenuSteps.java]: Done

File: src/test/java/stepDefinition/BookNowTopMenuSteps.java (hunk 3 above)
Comment (MlulekiN) [reply src/test/java/stepDefinition/BookNowTopMenuSteps.java]: Done

File: src/test/java/stepDefinition/MenuSteps.java [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+public class MenuSteps {
+    TestContext testContext;
+    WebDriver driver;
+
+    public MenuSteps(TestContext context) {
+        testContext = context;
+        driver = testContext.getWebDriverManager().getDriver();
+    }
+
```
Comment (benedictnkosi) [inline src/test/java/stepDefinition/MenuSteps.java:18]: add theUserClicksOnTheTopColumn to this class

File: src/test/java/stepDefinition/MenuSteps.java [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+    @Then("User validates content form is displayed")
```
Comment (benedictnkosi) [inline src/test/java/stepDefinition/MenuSteps.java:19]: this is not a menu step

File: src/test/java/pages/BookNowPage.java [hunk 6]
```diff
@@ -0,0 +1,7 @@
+package pages;
+
+import org.openqa.selenium.By;
+
+public class BookNowPage {
+    public static By contentForm = By.xpath("//div[@class='booking-form-content']");
```
Comment (benedictnkosi) [inline src/test/java/pages/BookNowPage.java:6]: call this booking form, and stop using content form. check other classes also

**Review by benedictnkosi — CHANGES_REQUESTED** [review]

**Pull request closed without merging**